TS/Tasks/CopyTasks.ts

```diff
@@ -0,0 +1,89 @@
+import { Helper, Config, BuildModes, creatGlob } from './../lib';
+import { TaskFunction, Globs, Gulp } from 'gulp';
+import { pipeline } from 'stream';
+import { isArray } from 'util';
+
+export namespace Copy {
+    import config = Config.Config;
+    import Folder = Config.Folder;
+    import Types = Config.Types;
+    import Static = Config.Static;
+    import ErrnoException = NodeJS.ErrnoException;
```

Comment:
Why is config lowercase and the other Uppercase?

Reply:
Ok, i understand the reason. A prefix like ImportConfig, ImportFolder, aso or something like this
`let Import = {};
import Import.Config = Config.Config;
import Import.Folder = Config.Folder;
`